docs/document/content/features/transaction/concept/base-transaction-seata.cn.md

```diff
@@ -13,4 +13,12 @@ weight = 3
 `Seata AT`事务模型包含TM(事务管理器)，RM(资源管理器)，TC(事务协调器)。其中TC是一个独立的服务需要单独部署，TM和RM以jar包的方式同业务应用部署在一起，它们同TC建立长连接，在整个事务生命周期内，保持RPC通信。
 其中全局事务的发起方作为TM，全局事务的参与者作为RM ; TM负责全局事务的begin和commit/rollback，RM负责分支事务的执行结果上报，并且通过TC的协调进行commit/rollback。
 
+Seata管理的分布式事务的典型生命周期：
+
+1. TM要求TC开始一个全新的全局事务。 TC生成一个代表该全局事务的XID。
```

Comment:
one more whitespace.

Reply:
what you mean one more whitespace . add after the colon ？

Reply:
between `。 TC`

Reply:
damn .... how do you find that. :sweat_smile: